Xamarin.Forms.Platform.iOS/ViewRenderer.cs

```diff
@@ -141,16 +141,18 @@ protected override void SetAccessibilityHint()
 
 			if (Element == null)
 				return;
+
+			var value = (string)Element.GetValue(AutomationProperties.HelpTextProperty) ?? _defaultAccessibilityHint;
```

Comment:
Hopefully my effort to make these generalized isn't more effort than it's worth :-) If Defs always make things fun 

The issue here is that value is being set prior to the *_defaultAccessibilityHint* being set

I could see it being useful to break that up into its own getter as well in case something else down the road wants to access Control's value.

Something like 
```C#
#if __MOBILE__
string ControlAccessibilityHint => Control.AccessibilityHint;
#else
string ControlAccessibilityHint => Control.AccessibilityTitle;
#endif
```
And then this section could be 

```C#
if (_defaultAccessibilityHint == null)
	_defaultAccessibilityHint = ControlAccessibilityHint ;

var helpTextValue = (string)Element.GetValue(AutomationProperties.HelpTextProperty) ?? _defaultAccessibilityHint;

#if __MOBILE__
			Control.AccessibilityHint = helpTextValue ;
#else
			Control.AccessibilityTitle = helpTextValue ;
#endif
```

And then something similar elsewhere.  It's nice when if defs can be pushed out to methods and then logic code can just not have them

Reply:
No limit to perfection. =)
If I develop this topic, then i can also add a setter. So it will be a little easier and removes some `__MOBILE__` directives.